mongorestore/mongorestore_test.go

```diff
@@ -27,10 +28,19 @@ func init() {
 	})
 }
 
-var (
-	testServer = "localhost"
-	testPort   = db.DefaultTestPort
-)
+func getRestoreWithArgs(additionalArgs ...string) (*MongoRestore, error) {
```

Comment:
All integration tests were rewritten to go through the entire options parsing path. It makes them easier to read and also enables testing of the actual user experience.